src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,49 @@
+package seedu.duke;
```

Comment:
All your classes and variables naming are intuitive and follows the coding standard. Good Job!